modules/test/playwright/tests/account-admin-web/accountOrganizations.spec.ts

```diff
@@ -340,10 +341,12 @@ test('LPD-47225 Can paginate organizations during assignment', async ({
 	await accountsPage.organizationsTab.click();
 	await accountOrganizationsPage.organizationsTable.newButton.click();
 
-	await setItemsPerPage(accountOrganizationSelectorPage.frame, 4);
+	page.waitForTimeout(500);
```

Comment:
can we do another expect of something in the page instead of the waitForTimeout?